node.gyp

```diff
@@ -971,7 +975,30 @@
             }]]
         }],
       ]
-    }
+    },
+    {
+      'target_name': 'node_postmortem_metadata',
+      'type': 'none',
+      'toolsets': ['host'],
+      'actions': [
+        {
+          'action_name': 'gen-postmortem-metadata',
+          'process_outputs_as_sources': 1,
+          'inputs': [
+            './tools/gen-postmortem-metadata.py',
```

Comment:
This is not enough to trigger a rebuild when object layouts change.  You need to take a dependency on the relevant headers in src/.